README.md

```diff
@@ -111,6 +111,39 @@ Example Usage:
 rudderClient.setAnonymousId(ANONYMOUS_ID);
 ```
 
+## Database Encryption
+
+To encrypt the database first add the below line to `import` the rudderClient and DBEncryption.
+
+```
+import rudderClient, { RUDDER_LOG_LEVEL, DBEncryption } from '@rudderstack/rudder-sdk-react-native';
+```
+
+Then open your `app/build.gradle` and add the dependency under `dependencies` as shown below:
+
+```
+//sql-cipher
+implementation "net.zetetic:android-database-sqlcipher:4.5.4"
+implementation "androidx.sqlite:sqlite:2.3.1"
+```
+
+Add the following code in your application:
+
+```
+const dbEncryption = new DBEncryption('versys', true);
+
+const config = {
+  dataPlaneUrl: <DATA_PLANE_URL>,
+  trackAppLifecycleEvents: true,
+  logLevel: RUDDER_LOG_LEVEL.DEBUG,
+  dbEncryption: dbEncryption,
```

Comment:
How they pass the encryption key?

Reply:
Refer to this section: https://github.com/rudderlabs/rudder-sdk-react-native/blob/feat/sdk-175-encrypt-values-in-sqlite3-database/README.md#database-encryption.

They first need to create the DBEncryption object:
```
const dbEncryption = new DBEncryption('versys', true);
````
Then pass that as a config param.